development-defaults.ini

```diff
@@ -10,10 +10,10 @@ consent_form_url = "http://magfest.org/minorconsentform"
 regdesk_email_signature = ''' - Victoria Earl,
 MAGFest Registration Chair'''
 
```

Comment:
This is a semi-related change which ought to be made in puppet... but this config option isn't puppetized.  Since the config stuff is changing when we move to Docker, I went ahead and just changed it here rather than putting in the work to puppetize yet another config option.